proto/xudrpc.proto

```diff
@@ -86,7 +86,13 @@ service Xud {
       get: "/v1/info"
     };
   }
-  
+
+  /*Get node reputation. */
```

Comment:
We can either drop this comment for now or change this to something like `/* Gets general information about a node. */`.